src/zoid/buttons/util.js

```diff
@@ -396,7 +396,7 @@ export const getModal: (
 
     return window[namespace].MessagesModal({
       account: `client-id:${clientID}`,
-      merchantId: merchantID?.join(",") || undefined,
+      merchantId: merchantID,
```

Comment:
For the modal input here we want this value to be the SDK script `merchantID` value because we don't want to pass in a `merchantId` value unless it is a partner integration, otherwise it may impact authentication.

Reply:
good callout, ill fix this one